examples/yaml-guestbook/index.ts

```diff
@@ -0,0 +1,6 @@
+// Copyright 2016-2018, Pulumi Corporation.  All rights reserved.
+
+import * as k8s from "@pulumi/kubernetes";
+
+// Create resources from standard Kubernetes guestbook YAML example.
+const guestbook = new k8s.Yaml("guestbook", { files: "yaml/*.yaml" });
```

Comment:
Two things:

1. Luke suggested the name, I don't like it either, but I couldn't think of anything better. I actually think that `Yaml` is a slightly better name than `yaml.Template`.
1. Is this enough of a problem to be a blocker for release?

Reply:
> Is this enough of a problem to be a blocker for release?

FWIW, it would kind of suck to immediately change the name of the thing in the next release, breaking anyone who used it (or immediately deprecating the name, which is what we'd probably do).